humanify/date.py

```diff
@@ -1,46 +1,41 @@
 import datetime as _datetime
 
 weekdays = [
-    'Monday',
-    'Tuesday',
-    'Wednesday',
-    'Thursday',
-    'Friday',
-    'Saturday',
-    'Sunday'
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
 ]
 
 months = [
-    'January',
-    'February',
-    'March',
-    'April',
-    'May',
-    'June',
-    'July',
-    'August',
-    'September',
-    'October',
-    'November',
-    'December'
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
 ]
 
-ordinals = [
-    'st',
-    'nd',
-    'rd',
-    'th'
-]
+ordinals = ["st", "nd", "rd", "th"]
 
 
 def date(input_datetime):
 
-    # Return if the type isn't correct
+    # Return if the type isn"t correct
```

Comment:
This is incorrect grammar

Reply:
He probably did replace every ' with a " using the CTR+F

Reply:
Yeah my bad, I'll fix

Reply:
Resolved with a32d2db